Front-end/parquick/src/contexts/UserContext.js

```diff
@@ -0,0 +1,7 @@
+//
+import { createContext } from "react";
+
+export const userContext = createContext({
```

Comment:
Great!
Normally we name context objects as if they were classes, so with camelcase (`UserContext`). But up to you.

We also tend to create helpers such as
```
function useUser() {
  return useContext(userContext);
}
```
so that you don't have to import `useContext` + `userContext` in every file you need the user data. But again it's a style nit.

Reply:
Applied